packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -516,7 +519,6 @@ export class PerformanceEvent {
         if (!this.event) {
             this.logger.sendErrorEvent({
                 eventName: "PerformanceEventAfterStop",
-                perfEventName: this.event!.eventName,
```

Comment:
This came from https://github.com/microsoft/FluidFramework/pull/2330, which is a massive PR so I didn't try to track it back through it.  Is there anything else we could put here instead to give a hint about what the logger's trying to log about? Some best-guess property from Props?

Reply:
@Vladsud

Reply:
We can record name of even, but that's probably enough - it it happens, than stream of events will likely contain enough clues to figure it out

Reply:
> We can record name of even[t], but that's probably enough

??? Where would we get that name?  That's the faulty code I'm deleting.

Also, how about `error.message` or something?

Reply:
In constructor - we do have event there, we just clear it after some event